src/main/java/com/exasol/bucketfs/UnsynchronizedBucket.java

```diff
@@ -79,4 +79,16 @@ void uploadStringContentNonBlocking(String content, String pathInBucket)
     // [impl->dsn~uploading-input-stream-to-bucket~1]
     void uploadInputStreamNonBlocking(Supplier<InputStream> inputStreamSupplier, String pathInBucket)
             throws BucketAccessException, TimeoutException;
+
+    /**
+     * Delete a file from BucketFS.
+     * <p>
+     * Warning: If you try to upload a file shortly (less than about 30s) after you deleted it, the upload will fail with access denied. (implementation details of BucketFS).
```

Comment:
I think here you can also use @implNote tag

Reply:
That made some trouble in the past, since it's not supported in all Javadoc versions.

Reply:
Remove the part in brackets. This does not add value here.